addons/website/static/tests/tours/website_text_font_size.js

```diff
@@ -88,6 +88,10 @@ function getFontSizeTestSteps(fontSizeClass) {
         }, {
             content: `Remove the text snippet containing the text with class ${fontSizeClass}`,
             trigger: `.oe_snippet_remove`,
+            async run(helpers) {
+                helpers.click();
+                await new Promise((r) => setTimeout(r, 300));
+            },
```

Comment:
Why do we need this? It's deserving a comment, but maybe it's going to be removed following the other review discussion

Reply:
I'll add a todo in a seperate commit